pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -453,11 +453,12 @@ type PullRequestGenerator struct {
 	GitLab          *PullRequestGeneratorGitLab          `json:"gitlab,omitempty" protobuf:"bytes,2,opt,name=gitlab"`
 	Gitea           *PullRequestGeneratorGitea           `json:"gitea,omitempty" protobuf:"bytes,3,opt,name=gitea"`
 	BitbucketServer *PullRequestGeneratorBitbucketServer `json:"bitbucketServer,omitempty" protobuf:"bytes,4,opt,name=bitbucketServer"`
+	Bitbucket       *PullRequestGeneratorBitbucket       `json:"bitbucket,omitempty" protobuf:"bytes,5,opt,name=bitbucket"`
```

Comment:
rename to `BitbucketCloud`?

Reply:
For some reason the SCM generator went with "bitbucket" instead of "bitbucketcloud," maybe because cloud was implemented first. I'll leave off the "cloud" for consistency. But this struct does need to be re-ordered to avoid breaking protobuf backwards compatibility.